README.md

```diff
@@ -41,7 +41,7 @@ The easiest and quickest way to setup your own routing engine backend is to use
 
 ### Using Docker
 
-We base the Docker images on Alpine Linux and make sure they are as lightweight as possible (around 10-15 MB).
+We base [our Docker images](https://hub.docker.com/r/osrm/osrm-backend/) on Alpine Linux and make sure they are as lightweight as possible (around 10-15 MB).
 In the following, replace `X.Y.Z` with the current stable release version.
```

Comment:
`vX.Y.Z` was also confusing, better to drop here any tags and use always implicitly default tag like `latest` or current stable tag. If someone knows about OSRM versions, it is not a problem to ask for a version tag or look at the OSRM docker page